file.js

```diff
@@ -4,7 +4,9 @@ const pino = require('./pino')
 const { once } = require('events')
 
 module.exports = async function (opts = {}) {
-  const destination = pino.destination({ dest: opts.destination || 1, sync: false })
+  const destOpts = { dest: opts.destination || 1, sync: false }
+  if (opts.mkdir) destOpts.mkdir = true
```

Comment:
why not add ternary condition on destOpts instead?

Reply:
I'm happy to do it anyway that is acceptable to the project. Adding an undefined property broke an existing test. I wrote it in such a way that the existing test would pass.